features/step_definitions/main_structure_steps.rb

```diff
@@ -76,6 +85,28 @@
   step("I wait for progress")
 end
 
+Then (/^I make sure "([^\"]*)" is not present$/) do |text|
+  if element_exists("* {text CONTAINS[c] '#{text}'}")
+    fail("Item %s should have been deleted" % [text])
+  end
+end
+
+Then (/^I press "([^\"]*)" button$/) do |text|
```

Comment:
I think this is defined by calabash already: https://github.com/calabash/calabash-android/blob/master/ruby-gem/lib/calabash-android/canned_steps.md#touching